### PR TITLE
Added and improved descriptions

### PR DESCRIPTION
**Pull request by 3ricsonn**
Besides an extra description to uninstall the program, I tried to make the usage description a little clearer as it was a bit confusing from my point of view.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,14 +21,26 @@ sudo make install
 make clean
 ```
 
+### Uninstalling 
+
+```
+cd /path/to/xwinwrap
+make
```
Comment (mmhobi7) [inline README.md:28]: why is this here?

File: README.md [hunk 2]
```diff
@@ -21,14 +21,26 @@ sudo make install
...
+sudo make uninstall
+make clean
```
Comment (mmhobi7) [inline README.md:30]: why are you making clean if you're about to delete the entire folder

File: README.md (hunk 1 above)
Comment (3ricsonn) [reply README.md:28]: If you refer to line 27, I use the "cd" command, because when the user opens for example a new terminal he must first go to the installation directory to uninstall the program.

File: README.md (hunk 2 above)
Comment (3ricsonn) [reply README.md:30]: You are right, this line can be deleted. My fault.
